scripts/local_testnet/README.md

```diff
@@ -5,58 +5,105 @@ This setup can be useful for testing and development.
 
 ## Requirements
 
-The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH`.
+The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH` (run `echo $PATH` to view all `PATH` directories).
```

Comment:
FYI `geth` and `bootnode`  just need to be available at the paths defined in `vars.env`, but yeah by default that would need to be on the $PATH, so we can leave this as it is to keep things simple.

Reply:
Thanks for the info, this is due to this?
https://github.com/sigp/lighthouse/blob/441fc1691b69f9edc4bbdc6665f3efab16265c9b/scripts/local_testnet/vars.env#L2-L3

Reply:
Yep!